rites/roman1969/src/locales/pt-br.ts

```diff
@@ -345,7 +362,7 @@ export const locale: Locale = {
       'Santos João de Brébeuf, Isaac Jogues, presbíteros, e Companheiros, mártires',
     john_de_britto_priest: 'São João de Brito, presbítero e mártir',
     john_eudes_priest: 'São João Eudes, presbítero',
-    john_fisher_bishop_and_thomas_more_martyrs: ' Santos João Fisher, bispo, e Tomás Moro, mártires',
+    john_fisher_bishop_and_thomas_more_martyrs: 'Santos João Fisher, bispo, e Tomás Moro, mártires',
```

Comment:
Missing source, a typo and there is an extra comma. Moreover, there is a difference betwee Ordo 2024 vs 2025 in the canonisation level (`São` + `Santo` vs `Santos`). Use one at your discretion.

```suggestion
    john_fisher_bishop_and_thomas_more_martyrs: 'Santos João Fisher, bispo e Tomás More, mártires', // src: arquidiocesemilitar.org.br/wp-content/uploads/2025/01/DIRETORIO-LITURGICO-OMB-2025-1_compressed.pdf#page=81
```

```suggestion
    john_fisher_bishop_and_thomas_more_martyrs: 'São João Fisher, bispo e Santo Tomás More, mártires', // src: https://www.scribd.com/document/697173100/Direto-rio-da-Liturgia-2024
```